messaging-activemq/src/main/java/org/wildfly/extension/messaging/activemq/jms/PooledConnectionFactoryService.java

```diff
@@ -520,17 +502,6 @@ private void configureCredential(List<ConfigProperty> properties) {
         }
     }
 
-    private void createJNDIAliases(final BindInfo bindInfo, List<String> aliases, ServiceController<ResourceAdapterDeployment> controller, ServiceTarget serviceTarget) {
-        for (final String alias : aliases) {
-            // do not install the alias' binder service if it is already registered
-            if (controller.getServiceContainer().getService(ContextNames.bindInfoFor(alias).getBinderServiceName()) == null) {
```

Comment:
This if test has been dropped. ExternalPooledConnectionFactoryService.createJNDIAliases has it too.

Reply:
this has been addressed